.gitignore

```diff
@@ -132,4 +132,4 @@ dmypy.json
 .pyre/
 
 scripts/minio-conf/
-scripts/minio-data/
+scripts/minio-data/
```

Comment:
nitpick - could you please remove this file completely from commit?

Reply:
Added a return character at the end :+1: